### DISCUSSION TITLE
Show package-support information for packages

### DISCUSSION THREAD

**Discussion by mhdawson** [npm/feedback#127] [Website]
The node.js package-maintenance team has defined [package-support](https://github.com/nodejs/package-maintenance/blob/main/docs/PACKAGE-SUPPORT.md) metadata which package authors can use to set expectations for consumers, and consumers can use to better understand the packages they depend on.

The team is looking at documenting how package authors can add this information to their README.md along these lines:

![image](https://user-images.githubusercontent.com/9373002/102649610-76f97a00-4137-11eb-868c-e61a1f34f0d3.png)

but it would be better/more effective in many cases if npm displayed this information on the website when people were searching for/looking at modules.  

There was a discussion in the last Node.js package maintenance meeting that we should suggest npm add that info, so here it is :)

**Comment by MylesBorins** [comment]
Hey @mhdawson nice seeing you here!

Is there a metadata format to define this support information?

**Reply by mhdawson** [reply]
Just follow the link [package-support](https://github.com/nodejs/package-maintenance/blob/main/docs/PACKAGE-SUPPORT.md)  from above its defined there.

**Reply by mhdawson** [reply]
unless you meant something more specific by "metadata format"

**Comment by mhdawson** [comment]
Just following up to see what next steps might be.

**Reply by MylesBorins** [reply]
To clarify... the specific ask is for npm to look for this package support meta data and in turn render information in the UI so that folks do not have to add badges?

**Reply by ljharb** [reply]
Yep! Ideally even that a "grayed out" support badge would be on every package, that would "light up" when the info was present - and clicking on the badge would display the relevant support info (or lack) with a link to docs and instructions of how to add/verify the info.
